_config.yml

```diff
@@ -160,3 +159,10 @@ fetch-remote:
       - dest: "registry"
         src:
           - "docs/configuration.md"
+
+  - repo: "https://github.com/crazy-max/buildkit" # FIXME: Use https://github.com/moby/buildkit
+    ref: "dockerfile-ref" # FIXME: Use master
```

Comment:
using my fork and keeping in draft for now while waiting for input of @thaJeztah.